src/components/SubHeader/SubHeader.js

```diff
@@ -0,0 +1,85 @@
+import "./SubHeader.css";
+import ratingIcon from "../../assets/icons/star.svg";
+import MenuFilter from "./Restaurant/MenuFilter";
+import BreadCrumb from "../../commonComponents/BreadCrumb";
+import getBreadCrumbConfig from "./helpers/breadCrumbConfig";
+import Offers from "./Restaurant/Offers";
+import Feedback from "./Restaurant/Feedback";
+import React from "react";
+import { getRestaurantDetails } from "../../apis/getRestaurantDetails";
+import Header from "./Restaurant/Header";
+import Spinner from "../../commonComponents/Spinner";
+
+class SubHeader extends React.Component {
+  state = {
+    restaurantDetails: {},
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    getRestaurantDetails(1)
+      .then((res) => res.json())
```

Comment:
make a rule of thumb and stop writing anonymous functions, always use named functions